Port.md

```diff
@@ -0,0 +1,57 @@
+# Ports
+
+## Status
+The status of ports is kept in the main module [README](README.md).
+
+## Overview
+ZikiChombo sio code has a porting process that is unconventional due to the
+nature sound support on host systems.  Usually, to port a system, on takes some
+desired cross-host functionality, such as a filesystem or user interface, and
+implements it the OS or OS/hardware combination of interest and that's that 
+(to make a long story short).
+
+For sound, the problem is that hosts usually have multiple different 
+software components as entry points to playing and capturing sound,
+and moreover these entry points have some mutually exclusive 
+properties, such as lower bounds on latency, availability of 
+duplex synchronised by hardware audio clock, ability to be shared
+accross many users or applications, etc.
+
+Although the main cpu hardware architectures can certainly be a concern,
```

Comment:
```diff
-cpu
+CPU
```

Reply:
thanks. 

First I need to make it at least compile and pass some tests.

Maybe by the end of the weekend, maybe during next week.